scripts/cnv_wdl/cnv_common_tasks.wdl

```diff
@@ -42,18 +44,22 @@ task CollectCoverage {
     Int? mem
 
     # If no padded target file is input, then do WGS workflow
-    Boolean is_wgs = select_first([padded_targets, ""]) == ""
+    Boolean is_wgs = !defined(padded_targets)
 
     # Sample name is derived from the bam filename
     String base_filename = sub(sub(sub(bam, "gs://", ""), "[/]*.*/", ""), "\\.bam$", "")
-
+ 
+    # Output file name depending on type of coverage
+    String cov_output_name = if (is_wgs && (select_first([transform, ""])  == "RAW")) then "${base_filename}.coverage.tsv.raw_cov" else "${base_filename}.coverage.tsv"
```

Comment:
It appears we populate our WDL scripts repo using this python script: https://github.com/broadinstitute/wdl/blob/develop/scripts/wrappers/gatk/gatkToWdlWrapper.py. At glance, it doesn't appear to require the javadoc portion of code, which is what I remember being the original target. Let's ask @knoblett to confirm. 

Since you are providing recommended workflows in WDL scripts within the repo, I think as long as these are up to date, and the documentation is up to date, the WDL repo translation is less of a concern as it lists all of the tool's arguments as task variables.

I think this is what you are asking? Or are you just letting me know of these changes? I can change these in the example commands given in the javadoc portion if needed. It's no problem.